templates/packer-ansible.dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rdouglass/packer-ansible:latest
```

Comment:
My concern with using this image is that I have no idea what it contains. Can we do one of the following here:

- Update the description of this image on [it's DockerHub page](https://hub.docker.com/r/rdouglass/packer-ansible)
- Choose an image that's better documented & supported

Reply:
We probably should more formalize a Dockerfile since I can't even find the Dockerfile I used to make `rdouglass/packer-ansible`. If I'm not mistaken it was a Ubuntu image that installed ansible and packer.